lib/binary_to_decimal.rb

```diff
@@ -6,4 +6,9 @@
 # the algorithm you devised in class.
 def binary_to_decimal(binary_array)
   raise NotImplementedError
+  decimal = 0
+  8.times do |i|
+    decimal += binary_array[i] * 2 ** (7 - i)
```

Comment:
Good job with using the math operator **. Can you think of a faster way to perform this without having to use the ** operation? Your code currently requires recalculating 2 to the power of n (number of bits) at every iteration.